packages/ez_api/ez_api.1.1.1/opam

```diff
@@ -0,0 +1,75 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Easy API library and tools"
+maintainer: ["Fabrice Le Fessant <fabrice.le_fessant@ocamlpro.com>"]
+authors: [
+  "Cagdas Bozman <cagdas.bozman@ocamlpro.com>"
+  "Michael Laporte <michael.laporte@ocamlpro.com>"
+  "Fabrice Le Fessant <fabrice.le_fessant@ocamlpro.com>"
+  "Maxime Levillain <maxime.levillain@ocamlpro.com>"
+  "Alain Mebsout <alain.mebsout@ocamlpro.com>"
+]
+license: "LGPL-2.1-only with OCaml-LGPL-linking-exception"
+homepage: "https://github.com/ocamlpro/ez_api"
+doc: "https://ocpmax.github.io/ez_api/ez_api/index.html"
+bug-reports: "https://github.com/ocamlpro/ez_api/issues"
+depends: [
+  "dune" {>= "2.8"}
+  "ocaml" {>= "4.08"}
+  "json-data-encoding" {>= "0.7.0"}
+  "lwt" {>= "4.0.0"}
+  "ezjsonm" {>= "1.0.0"}
```

Comment:
```suggestion
  "ezjsonm" {>= "1.1.0"}
```

possibly more of the fixes added in https://github.com/ocaml/opam-repository/pull/20150 are needed